imports/plugins/core/shipping/client/containers/parcelSizeSettings.js

```diff
@@ -0,0 +1,117 @@
+import { Meteor } from "meteor/meteor";
+import { registerComponent, composeWithTracker } from "@reactioncommerce/reaction-components";
+import { Reaction, i18next, Logger } from "/client/api/";
+import { Shops } from "/lib/collections";
+import { ShippingParcel } from "/lib/collections/schemas";
+import { convertWeight, convertLength } from "/lib/api";
+import ParcelSizeSettings from "../components/parcelSizeSettings";
+
+/**
+ * @method saveDefaultSize
+ * @summary call "shipping/updateParcelSize" method
+ * @param {Object} size - size object to be saved
+ * @since 1.1.12
+ * @return {Function} callback
+*/
+const saveDefaultSize = (size) => {
+  const parcel = ShippingParcel.clean(size);
+  Object.keys(parcel).forEach((key) => {
+    parcel[key] = Number(parcel[key]);
+  });
```

Comment:
Casting parcel values to `Number` should not be necessary here because `.clean` should be doing it.